modules/asset-manager-storage-aws/src/main/java/org/opencastproject/assetmanager/aws/AwsAbstractArchive.java

```diff
@@ -32,6 +32,7 @@
 import org.opencastproject.assetmanager.aws.persistence.AwsAssetDatabaseException;
 import org.opencastproject.assetmanager.aws.persistence.AwsAssetMapping;
 import org.opencastproject.assetmanager.impl.VersionImpl;
+import org.opencastproject.assetmanager.impl.persistence.Database;
```

Comment:
This dependency seems to be causing wiring issues

```
org.apache.felix.resolver.reason.ReasonException: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=opencast-allinone; type=karaf.feature; version="[14.0.0.SNAPSHOT,14.0.0.SNAPSHOT]"; filter:="(&(osgi.identity=opencast-allinone)(type=karaf.feature)(version>=14.0.0.SNAPSHOT)(version<=14.0.0.SNAPSHOT))" [caused by: Unable to resolve opencast-allinone/14.0.0.SNAPSHOT: missing requirement [opencast-allinone/14.0.0.SNAPSHOT] osgi.identity; osgi.identity=opencast-asset-manager-storage-aws; type=osgi.bundle; version="[14.0.0.SNAPSHOT,14.0.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve opencast-asset-manager-storage-aws/14.0.0.SNAPSHOT: missing requirement [opencast-asset-manager-storage-aws/14.0.0.SNAPSHOT] osgi.service; filter:="(objectClass=org.opencastproject.assetmanager.impl.persistence.Database)"; effective:=active]]
```
Missing export from asset-manager-impl bundle?

Reply:
Yep.  I assumed that the AM Database was a service.  It's not.  It's a... thing.  I've got it worked around by replacing this reference with the AM itself, which then exposed a more serious issue I haven't had time to deal with yet.  Will ping you when I get around to looking at this again.